Source/santad/DataLayer/SNTEventTable.mm

```diff
@@ -73,6 +73,15 @@ - (uint32_t)initializeDatabase:(FMDatabase *)db fromVersion:(uint32_t)version {
     newVersion = 4;
   }
 
+  if (version < 5) {
+    // Rename the filesha256 column to uniqueid because different stored event types
+    // contain different content for determining uniqueness.
+    [db executeUpdate:@"ALTER TABLE events RENAME COLUMN filesha256 TO uniqueid"];
+    [db executeUpdate:@"DROP INDEX filesha256"];
```

Comment:
```suggestion
    [db executeUpdate:@"DROP INDEX filesha256"];
    [db executeUpdate:@"ALTER TABLE events RENAME COLUMN filesha256 TO uniqueid"];
```

Reply:
Yeah, just making the output of `.schema` make more sense, not strictly necessary.